Orange/data/io.py

```diff
@@ -626,43 +651,46 @@ class BasketFormat(FileFormat):
     EXTENSIONS = ('.basket', '.bsk')
     DESCRIPTION = 'Basket file'
 
-    @classmethod
-    def read_file(cls, filename, storage_class=None):
-        from Orange.data import _io, Table, Domain
-        import sys
-        if storage_class is None:
-            storage_class = Table
+    def read(self):
+        if self.wrapper is None or self.wrapper is _IDENTITY:
+            self.wrapper = Table
```

Comment:
I think we should consider throwing these wrappers out. Is `wrapper` ever not `Table`? If it isn't then the caller is obviously aware of it and can invoke `wrapper(reader.read())` on their end.